cfgmgr/buffermgrd.cpp

```diff
@@ -31,6 +31,7 @@ using json = nlohmann::json;
 int gBatchSize = 0;
 bool gSwssRecord = false;
 bool gLogRotate = false;
+bool gExitInSaiFailure = true;
```

Comment:
Why do we have this Sai failure in manager daemons?

Reply:
This is because the orch class is used in the manager daemons. Thus, the global variables need to be defined here to avoid compilation errors. Here is an existing comment in the manager daemons that talks about why it is needed here.
```
/*
 * Following global variables are defined here for the purpose of
 * using existing Orch class which is to be refactored soon to
 * eliminate the direct exposure of the global variables.
 *
 * Once Orch class refactoring is done, these global variables
 * should be removed from here.
 */
```